include/aos/common/pkcs11/pkcs11.hpp

```diff
@@ -656,6 +656,15 @@ class PrivateKey {
  */
 class Utils {
 public:
+    /**
+     * Maximum size of allocator's memory that is supposed to be reserved for local objects.
+     * Required by some Utils functions like GenerateRSA(ECDSA)KeyPairWithLabel, etc.
+     */
```

Comment:
Need mention that additionally required pkcs11::cPrivateKeyMaxSize * num keys for load key API

Reply:
fixed